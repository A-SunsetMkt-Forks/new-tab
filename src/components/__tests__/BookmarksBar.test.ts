import BookmarksBar from '../BookmarksBar.svelte';

describe('BookmarksBar component', () => {
  it('renders correctly', () => {
    expect.assertions(1);
    const target = document.createElement('div');
    new BookmarksBar({ target });
    expect(target.innerHTML).toMatchSnapshot();
  });
});
