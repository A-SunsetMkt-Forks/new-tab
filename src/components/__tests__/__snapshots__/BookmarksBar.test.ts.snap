// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BookmarksBar component renders correctly 1`] = `"<div class=\\"bookmarks\\">  </div>"`;
